PINCache/PINOperationQueue.m

```diff
@@ -98,6 +105,7 @@ - (instancetype)initWithMaxConcurrentOperations:(NSUInteger)maxConcurrentOperati
     _highPriorityOperations = [[NSMutableOrderedSet alloc] init];
     
     _referenceToOperations = [NSMapTable weakToWeakObjectsMapTable];
+    _identifierToOperations = [NSMapTable weakToWeakObjectsMapTable];
```

Comment:
Is it safe for this to be weakToWeak? I think it may need to be weakToStrong, as we don't know if callers of the api will keep strong references to their identifiers (i.e. they may just pass in a locally allocated NSString which could fall out of scope).

I think that should be ok though because they'll fall out when the identifier gets released?